### DISCUSSION TITLE
Seeing no records in /prisma app

### DISCUSSION THREAD

**Discussion by james0r** [sidebase/sidebase#97] [Q&A] [answered]
Just installed using the modular option in the wizard and ran it in dev mode. Should I be seeing records on the `/prisma` route? 
<img width="574" alt="image" src="https://user-images.githubusercontent.com/34731936/215383865-5ce71da4-8d7d-4b44-a00c-d55fe616c8e4.png">

**Comment by BracketJohn** [comment, marked as answer]
Hey @james0r 👋 

That is indeed expected: When you first setup the app and databse, the database is empty, so 0 records are returned.

You can for example create a POST endpoint in order to create records (:

**Reply by james0r** [reply]
Thanks! Learning a bit about seeding with Prisma now. https://www.prisma.io/docs/guides/database/seed-database?query=prisma%20&page=1

**Reply by BracketJohn** [reply]
sounds great (: Feel free to share what you're building, either here or on discord: https://discord.gg/NDDgQkcv3s
